template/v2/Dockerfile

```diff
@@ -57,10 +59,13 @@ RUN apt-get update && apt-get upgrade -y && \
     touch /etc/krb5.conf.lock && chown ${NB_USER}:${MAMBA_USER} /etc/krb5.conf* && \
     # Note that we do NOT run `rm -rf /var/lib/apt/lists/*` here. If we did, anyone building on top of our images will
     # not be able to run any `apt-get install` commands and that would hamper customizability of the images.
-    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
-    unzip awscliv2.zip && \
+    curl -O "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" && \
+    curl -O "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip.sig" && \
+    gpg --import /tmp/aws-cli-public-key.asc && \
```

Comment:
Set an explicit trust level for the imported key:

Ex.
```
gpg --import /tmp/aws-cli-public-key.asc && \
 echo "trust\n5\ny\n" | gpg --command-fd 0 --edit-key "aws-cli@amazon.com" && \
```

5 is the number for ultimate trust. Levels include:
• **Unknown** (default when imported)
• **Never trust**
• **Marginal trust**
• **Full trust**
• **Ultimate trust**